metadata-ingestion/tests/integration/mode/test_mode.py

```diff
@@ -12,6 +12,7 @@
 FROZEN_TIME = "2021-12-07 07:00:00"
 
 JSON_RESPONSE_MAP = {
+    "https://app.mode.com/api/verify": "verify.json",
```

Comment:
is this file missing?

Reply:
Interesting, it seems like it never was part of the test resources

Reply:
I found it in the meantime :)